service/src/main/java/bio/terra/drshub/services/DrsProviderService.java

```diff
@@ -61,7 +70,13 @@ public UriComponents getUriComponents(String drsUri) {
       }
     } else if (hostNameMatch.find(0)) {
       dnsHost = hostNameMatch.group("hostname");
-      strippedPath = drsUri.replaceAll("(?:dos|drs)://", "").replace(dnsHost + "/", "");
+      strippedPath =
+          drsUri
+              .replaceAll("(?:dos|drs)://", "")
+              .replace(dnsHost + ":", "")
+              .replace(dnsHost + "/", "");
```

Comment:
Is adding the .replace(dnsHost + ":", "") necessary? we are thinking no but I added just in case.

Reply:
What's the use case for this extra `replace`?